UKHomeOffice/nmsw-ui
# Test nmsw 460

- Moved Fal-1 test files under Fal1 folder
- Refactored the code for verifying ship name and voyage type
- Added test to verify check-your-answers-page feature

## To test
```
npm run cypress:open:dev -- --env MAIL_API_KEY=<API_KEY>
```
and select check-your-answers-page.feature
